src/DependencyInjection/Compiler/ConfigParserPass.php

```diff
@@ -177,6 +177,7 @@ private function mappingConfig(array $config, ContainerBuilder $container): arra
 
         // app only config files (yml or xml or graphql)
         if ($mappingConfig['auto_discover']['root_dir'] && $container->hasParameter('kernel.root_dir')) {
```

Comment:
Why this change in this PR? Please open a dedicated PR for this.

Reply:
https://github.com/overblog/GraphQLBundle/pull/816